src/ILVerify/tests/BasicArithmeticTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Internal.IL;
+using Internal.TypeSystem;
+using Internal.TypeSystem.Ecma;
+using Xunit;
+
+namespace ILVerify.Tests
+{
+    /// <summary>
+    /// Tests for BasicArithmeticTests.il
+    /// </summary>
+    public class BasicArithmeticTests
+    {
+        public static EcmaModule s_testModule = TestDataLoader.GetModuleForTestAssembly("BasicArithmeticTests.dll");
+
+        [Fact]
+        public void SimpleAddTest()
+        {
+            ILImporter importer = TestDataLoader.GetMethodDesc(s_testModule, "[BasicArithmeticTests]BasicArithmeticTestsType.ValidSimpleAdd");
+            
```

Comment:
I think this should be `[Theory]`, the test data should be auto-generated by enumerating all methods in the test file, and filtering the ones for given theory based on the test name conventions - e.g. one theory should get all valid tests and other theory should get all invalid tests. For invalid tests, you can also have convention to encode the expected error in the test name.